core/templates/components/rubrics-editor/rubrics-editor.component.spec.ts

```diff
@@ -93,6 +93,31 @@ describe('Rubrics Editor Component', () => {
       .toBeTrue();
   });
 
+  it('should check for maximum character length of explanation', () => {
```

Comment:
The test description can be improved. Please read the [wiki](https://github.com/oppia/oppia/wiki/Frontend-tests#write-frontend-tests) once to understand how to write FE Unit tests with proper conventions.

Reply:
Sure, thanks for your response.